CHANGELOG.md

```diff
@@ -90,13 +90,15 @@ the same every time it is rendered, we now warn if it is missing.
 #### General
 - Add WGSL examples to complement existing examples written in GLSL by @norepimorphism in [#2888](https://github.com/gfx-rs/wgpu/pull/2888)
 
-### Performance
```

Comment:
Where did the documentation tag go?

Reply:
There were two `### Documentation` sections. I merged them.